templates/zilencer/upgrade.html

```diff
@@ -43,14 +43,14 @@ <h3>{{ _("Payment schedule") }}</h3>
                             </div>
                         </label>
                         <label>
-                            <input type="radio" name="plan" value="{{ nickname_monthly }}" />
+                            <input type="radio" name="plan" value="{{ nickname_monthly }}" data-amount="{{ monthly_plan_amount }}" />
                             <div class="box">
                                 <div class="schedule-time">{{ _("Pay monthly") }}</div>
                                 <div class="schedule-amount">${{ monthly_plan_amount }}/user/month</div>
                             </div>
                         </label>
                     </div>
-                    <p>You&rsquo;ll initially be charged <b><span id="charged_amount">XXX</span></b> for <b>{{ seat_count }}</b> users. You&rsquo;ll receive prorated charges and credits as users are added, deactivated, or become inactive.</p>
+                    <p>You&rsquo;ll initially be charged <b><span id="charged_amount">{{ annual_plan_amount * seat_count }}</span>$</b> for <b>{{ seat_count }}</b> users. You&rsquo;ll receive prorated charges and credits as users are added, deactivated, or become inactive.</p>
```

Comment:
ah wow, I didn't realize you could multiply inside of the `{{  }}`!
A minor thing is the `$` should go at the beginning. (Possibly it should be written as `$50 USD`, i.e. with the USD?)